src/status_im/ui/screens/wallet/send/views/amount.cljs

```diff
@@ -0,0 +1,314 @@
+                                          :on-press       #(re-frame/dispatch
+                                                            [:navigate-to :wallet-choose-asset
+                                                             {:on-asset (fn [{:keys [symbol]}]
+                                                                          (when symbol
+                                                                            (if-not (= symbol (:symbol @tx-atom))
+                                                                              (update-amount-field nil))
+                                                                            (swap! tx-atom assoc :symbol symbol)
+                                                                            (common/refresh-optimal-gas web3 tx-atom))
+                                                                          (re-frame/dispatch [:navigate-back]))}])
+                                          :underlay-color colors/white-transparent}
+               [show-current-asset coin]]
+              [react/view {:flex 1}
+               [react/view {:flex 1}]
+               [react/view {:justify-content :center
+                            :align-items     :center
+                            :flex-direction  :row}
+                (when error-message
+                  [tooltip/tooltip error-message {:color        colors/white
+                                                  :font-size    12
+                                                  :bottom-value 15}])
+                [react/text-input
+                 {:ref                    #(reset! amount-input-ref %)
+                  :on-change-text         update-amount-field
+                  :keyboard-type          :numeric
+                  :accessibility-label    :amount-input
+                  :auto-focus             true
+                  :auto-capitalize        :none
+                  :auto-correct           false
+                  :placeholder            "0"
+                  :placeholder-text-color colors/blue-shadow
+                  :multiline              true
+                  :max-length             20
+                  :default-value          input-amount
+                  :selection-color        colors/green
+                  :keyboard-appearance    :dark
+                  :style                  {:color               colors/white
+                                           :font-size           30
+                                           :font-weight         "500"
+                                           :padding-horizontal  10
+                                           :padding-vertical    7
+                                           :max-width           290
+                                           :text-align-vertical :center}}]
+                [react/text {:style {:color               (if (not (string/blank? input-amount))
```

Comment:
not of great importance again, but we could avoid using `not` if by swapping colors